.circleci/config.yml

```diff
@@ -6,5 +6,25 @@ jobs:
     steps:
       - checkout
       - run:
-          name: Run tests
+          name: Prepare Python env # install python3.7, python3-pip and pipenv for pyspark.
```

Comment:
Can we try to build a custom image to speed up the tests? https://circleci.com/docs/2.0/custom-images/

Reply:
on my way